test/functional/ui/mouse_spec.lua

```diff
@@ -617,7 +635,7 @@ describe('ui/mouse/input', function()
     feed('<cr>')
   end)
 
-  it('mouse whell will target the hovered window', function()
+  local function wheel(use_meth)
```

Comment:
`use_api` ? Would like to move away from referring to the API as "methods", I don't think that makes sense to newcomers.